pkgs/applications/video/obs-studio/wlrobs.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchhg, obs-studio, wayland }:
+
+stdenv.mkDerivation rec {
+  pname = "wlrobs";
+  version = "82e2b93c6f662dfd9d69f7826c0096bef585c3ae";
+  src = fetchhg {
+    url = "https://hg.sr.ht/~scoopta/wlrobs";
+    rev = "${version}";
+    sha256 = "1d2mlybkwyr0jw6paamazla2a1cyj60bs10i0lk9jclxnp780fy6";
+  };
+  buildInputs = [ obs-studio wayland ];
+  buildPhase = ''
+    cd Release
+    make
+  '';
+  installPhase = ''
+    PLUGIN_FOLDER="$out/share/obs/obs-plugins/wlrobs/bin/64bit"
+    mkdir -p "$PLUGIN_FOLDER"
+    cp libwlrobs.so "$PLUGIN_FOLDER"
```

Comment:
Consider using the ``install`` command.

Reply:
Admittedly, I am not sure what this does for this particular case, but fixed.

Reply:
The replacement for the three lines would be `install -D libwlrobs.so $out/share/obs/obs-plugins/wlrobs/bin/64bit/libwlrobs.so`